projects/packages/action-bar/src/class-action-bar.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Put your classes in this `src` folder!
+ *
+ * @package automattic/jetpack-action-bar
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Action_Bar class.
+ */
+class Action_Bar {
+	/**
+	 * Enqueue scripts for rendering Action Bar client.
+	 */
+	public function enqueue_scripts() {
+		if ( is_admin() || ! is_single() ) {
+			return;
+		}
+
+		Assets::register_script(
+			'jetpack-action-bar',
+			'build/action-bar.js',
+			dirname( __DIR__ ) . DIRECTORY_SEPARATOR . 'src', // A full path to a file or a directory inside a plugin.
```

Comment:
Shouldn't this look for the file one level up? I may be missing something here though.

```suggestion
			'../build/action-bar.js',
			__FILE__,
```

Reply:
Yes, that works, thanks!